packages/babel-preset-fbjs/plugins/__tests__/inline-requires-test.js

```diff
@@ -1,180 +1,79 @@
-      'function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }',
-      'console.log(_foo.default);',
-    ]);
-  });
-
-  it('should be compatible with `transform-modules-commonjs` when using named imports', function() {
-    compare([
-      'import {a} from "./a";',
-      'var D = {',
-      '  b: function(c) { c ? a(c.toString()) : a("No c!"); },',
-      '};',
-    ], [
-      'var D = {',
-      '  b: function (c) {',
-      '    c ? (0, require("./a").a)(c.toString()) : (0, require("./a").a)("No c!");',
-      '  }',
-      '};',
-    ]);
-  });
-
-  it('should inline the given method calls', function() {
-    compare([
-      'const inlinedCustom = customStuff("foo");',
-      'const inlinedRequire = require("bar");',
-      '',
-      'inlinedCustom();',
-      'inlinedRequire();',
-    ], [
-      'customStuff("foo")();',
-      'require("bar")();',
-    ], {
-      inlineableCalls: ['customStuff'],
-    });
-  });
-
-  it('should remove loc information from nodes', function() {
-    var ast = transform(['var x = require("x"); x']).ast;
-    var expression = ast.program.body[0].expression;
-
-    function noLoc(node) {
-      expect(node.start).toBeUndefined();
-      expect(node.end).toBeUndefined();
-      expect(node.loc).toBeUndefined();
-    }
-
-    noLoc(expression);
-    noLoc(expression.arguments[0]);
-  });
+const inlineRequiresPlugin = require('../inline-requires');
+const pluginTester = require('babel-plugin-tester');
+
+pluginTester({
+  plugin: inlineRequiresPlugin,
+  pluginOptions: {
+    ignoredRequires: ['Bar'],
+    inlineableCalls: ['customRequire'],
+  },
+  tests: {
+    'inlines `Foo`': {
+      code: "const Foo = require('Foo'); Foo;",
+      snapshot: true,
+    },
+
+    'inlines `Foo` multiple times': {
+      code: "const Foo = require('Foo'); Foo; Foo;",
+      snapshot: true,
+    },
+
+    'inlines `Foo` in multi-variable declarations': {
+      code: "const Foo = require('Foo'), Bar = 123, Baz = 456; Foo;",
+      snapshot: true,
+    },
+
+    'ignores `Foo` when not assigned to a variable': {
+      code: "require('Foo');",
+      snapshot: true,
+    },
+
+    'deletes `Foo` if unused': {
+      code: "const Foo = require('Foo');",
+      snapshot: true,
+    },
+
+    'ignores `Foo` when not assigned to a variable': {
+      code: "const Foo = require('Foo'); Foo; Foo;",
+      snapshot: true,
+    },
+
+    'inlines `Foo` properties': {
+      code: "const Baz = require('Foo').Baz; Baz;",
+      snapshot: true,
+    },
+
+    'inlines `Foo` properties with destructuring': {
+      code: "const {Baz} = require('Foo'); Baz;",
+      snapshot: true,
+    },
+
+    'inlines `Foo` before it is declared': {
+      code: "const doSomething = () => Foo; const Foo = require('Foo');",
+      snapshot: true,
+    },
+
+    'inlines `Foo` when required via `customRequire`': {
+      code: "const Baz = require('Bar').Baz; Baz;",
```

Comment:
To make this test match the description shouldn't the code be `const Baz = customRequire('Bar').Baz; Baz;`?